.github/workflows/dependency-graph.yml

```diff
@@ -0,0 +1,16 @@
+# .github/workflows/dependency-graph.yml
+name: Submit Dependency Graph
+on:
+  push:
+    branches: [1.10.x] # default branch of the project
```

Comment:
Btw I am not familiar with how Dependency Graph submission API works, but I guess once the PR is approved, we need to forward port it to develop branch with `branches: [developed]`?